README.md

```diff
@@ -42,6 +42,7 @@ Optional:
 
 * schema=`default schema`
 * timezone=`default timezone`
+* 'nuodb6' instead of 'nuodb' for ipv6 when broker_address is a hostname
```

Comment:
What about this, which to me seems like a more conventional DSN

nuodb://username:password@broker_address/database?schema=schema_name&preferred_address_family=ipv6

Reply:
Done